lib/scala/logging-utils/src/main/java/org/enso/logger/LoggerUtils.java

```diff
@@ -0,0 +1,12 @@
+package org.enso.logger;
+
+public class LoggerUtils {
```

Comment:
Isn't the code a bit _over-modularized_? Whey do we have a `logging-utils` project and also `logging-jutil` project? Why `LoggerUtils` aren't merged with `Converter`?